src/glabs-left/web/index.scss

```diff
@@ -0,0 +1,95 @@
+@import '_core';
+
+a {
+  text-decoration: none;
+}
+
+h1, p {
+  margin: 0;
+}
+
+.container {
+  font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
```

Comment:
Nit (single quotes and spacing):

`  font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;`